test-case/check-capture.sh

```diff
@@ -57,6 +57,7 @@ loop_cnt=${OPT_VAL['l']}
 out_dir=${OPT_VAL['o']}
 file_prefix=${OPT_VAL['f']}
 
+start_test
```

Comment:
All of this churn is to keep the `systemctl_show_pulseaudio` in tools/kmod/sof_remove.sh instead of just dropping it from the sof_remove.sh along with sourcing the case-lib/lib.sh and be done?

Reply:
> instead of just dropping it from the sof_remove.sh along with sourcing the case-lib/lib.sh 

The (non-mandatory) pulseaudio check is debatable. Including `lib.sh` is much less so. There's no reason to copy/paste routines when 

Anyway this most important and urgent benefit of this design fix is preparing the fix for #1036 and #1112. The same design fix just happens to help with some other scripts like sof_remove.sh.